### DISCUSSION TITLE
$pinia is not in the context inside a nuxt 2 plugin

### DISCUSSION THREAD

**Discussion by TimvdEijnden** [vuejs/pinia#2017] [Help and Questions]
### Reproduction

https://stackblitz.com/edit/nuxt-starter-82eu7r?file=README.md,plugins%2Finit-flag.js,package.json,pages%2Findex.vue,nuxt.config.js

### Steps to reproduce the bug

- Run the app using `nuxt dev`
- Check the console.log

### Expected behavior

$pinia should be in the list of keys

### Actual behavior

$pinia is missing in the list of keys.

### Additional information

We're moving away from vueX, and there used to be a $store in the context. But now with trying to use Pinia it isn't there.

**Comment by TimvdEijnden** [comment]
Never mind, I didn't notice this warning:

```js
WARN  Module pinia is disabled due to incompatibility issues
- [bridge] Nuxt bridge is required
```

**Comment by posva** [comment]
You seem to be using the wrong version of the plugins. See https://pinia.vuejs.org/ssr/nuxt.html#nuxt-2-without-bridge

You can check [this example](https://github.com/piniajs/example-nuxt-2) for a complete working sample

Although the recommended approach would be to use Nuxt bridge so smoothen up the migration to Nuxt 3 😉 . That version would also support a more up to date pinia plugin
